.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,9 @@
 <!--- Provide a general summary of your changes in the Title above -->
+<!--- Please be aware that as of the 31st January 2022 we no longer support 7.10.x.
+New PRs to hotfix-7.10.x will be invalid. If your fix is still applicable to 7.12.x, 
+please create the pull request to the hotfix branch accordingly. -->
+<!--- When creating a pull request please ensure that you are following the standard for commit messages-->
```

Comment:
This is not relevant to all commit, and should be part of documentation we reference separately from this change